src/sentry/integrations/github/blame.py

```diff
@@ -95,9 +95,7 @@ def create_blame_query(file_path_mapping: FilePathMapping, extra: Mapping[str, A
 
     query = f"""query {{{repo_queries}\n}}"""
 
-    logger.info(
-        "get_blame_for_files.create_blame_query.created_query", extra={**extra, "query": query}
-    )
+    logger.info("get_blame_for_files.create_blame_query.created_query", extra=extra)
```

Comment:
Removed from logs since its a formatted graphql query which is extremely difficult to parse and introduces a lot of noise.